docs/guides-blog.md

```diff
@@ -32,7 +32,7 @@ authorFBID: 503283835
 title: Introducing Docusaurus
 ---
 
-Lorem Ipusm..
+Lorem Ipusm...
```

Comment:
Since you're touching this line, could you correct it to Ipsum? 😂

Reply:
fixed